src/test/net/DatabaseDownloader_test.cpp

```diff
@@ -139,18 +161,12 @@ class DatabaseDownloader_test : public beast::unit_test::suite
             std::function<void(boost::filesystem::path)>{std::ref(cb)});
         if (!BEAST_EXPECT(stat))
         {
-            log << "Failed. LOGS:\n" + downloader.sink_.messages().str();
-            log << "DownloadCompleter::waitComplete timed out. "
-                   "DatabaseDownloader session active? "
-                << std::boolalpha << downloader->sessionIsActive()
-                << " DatabaseDownloader is stopping? " << std::boolalpha
-                << downloader->isStopping() << std::endl;
-
+            log << "Failed. LOGS:\n" + dl.sink_.messages().str();
```

Comment:
This is now the only write in this file going to `log`, the rest are going to `fail` messages.  And it lost much of what was being reported before.  Is all of this intentional?

Reply:
This isn't a `Downloader` callback failure, so I wasn't compelled to provide the additional log information.

Reply:
Made the failure reporting function more generic and included it here as well. Done